homeworks/sasha.demidenko_SashaDemi/GamesProject/Games.css

```diff
@@ -0,0 +1,106 @@
+.main {
+  /*display: flex;
```

Comment:
Please remove the commented code.
Please fix page according to design:
- font sizes
- margins
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/91f8fa7d-38a5-4404-8ec5-8843004f76dc)

Reply:
Please fix this alignment in your next task
<img width="552" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/8863af2e-f915-41ff-975c-81efef761dbb">